bucket/webtorrent.json

```
@@ -5,15 +5,26 @@
     "license": "MIT",
     "architecture": {
         "64bit": {
-            "url": "https://github.com/webtorrent/webtorrent-desktop/releases/download/v0.20.0/WebTorrent-0.20.0-full.nupkg#/dl.7z",
+            "url": "https://github.com/webtorrent/webtorrent-desktop/releases/download/v0.20.0/WebTorrent-0.20.0-full.nupkg",
             "hash": "sha1:2CFE5124C24E70CC0BAFF535BA929F77A3723A29"
         },
         "32bit": {
-            "url": "https://github.com/webtorrent/webtorrent-desktop/releases/download/v0.20.0/WebTorrent-0.20.0-ia32-full.nupkg#/dl.7z",
+            "url": "https://github.com/webtorrent/webtorrent-desktop/releases/download/v0.20.0/WebTorrent-0.20.0-ia32-full.nupkg",
             "hash": "sha1:21B1CC158D4035DEEBCDC5B08DB8F2D4A37DC238"
         }
     },
     "extract_dir": "lib\\net45",
+    "pre_install": [
+        "if (!(Test-Path \"$persist_dir\\Portable Settings\")) {",
+        "    New-Item \"$dir\\Portable Settings\\Downloads\" -ItemType 'Directory' -Force | Out-Null",
+        "    foreach ($data in @(\"$env:Appdata\\WebTorrent\", \"$env:LocalAppdata\\WebTorrent\")) {",
```

Copy variable is just for keeping copy item call shorter

```suggestion
        "    $copy = \"$env:LOCALAPPDATA\\Webtorrent\\*\", \"$env:APPDATA\\Webtorrent\\*\"",
        "    Copy-Item $copy \"$dir\\Portable Settings\" -ErrorAction SilentlyContinue -Force -Recurse",
```